cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/query/IndexCreator.java

```diff
@@ -351,7 +352,7 @@ private String createIndexIndexStatementForIndex(String indexName, List<String>
         Joiner joiner = Joiner.on(",").skipNulls();
         String cols = joiner.join(columns);
 
-        return String.format("CREATE INDEX %s ON %s ( %s )", sqlIndexName, tableName, cols);
+        return String.format("CREATE INDEX \"%s\" ON \"%s\" ( %s )", sqlIndexName, tableName, cols);
```

Comment:
This should use a specific `Locale` for the `String.format()` rather than the default.